src/core/js/a11y/deprecated.js

```diff
@@ -0,0 +1,174 @@
+define([
+    'core/js/adapt'
+], function(Adapt) {
+
+    /**
+     * The old API is rerouted to the new API with warnings.
+     */
+
+    _.extend($.fn, {
+
+        isFixedPostion: function() {
+            Adapt.a11y.log.removed('$("..").isFixedPostion was unneeded and has been removed, let us know if you need it back.');
+            return false;
+        },
+
+        a11y_aria_label: function() {
+            Adapt.a11y.log.removed('$("..").a11y_aria_label was incorrect behaviour.');
+            return this;
+        },
+
+        limitedScrollTo: function() {
+            Adapt.a11y.log.removed('$.limitedScrollTo had no impact on the screen reader cursor.');
+            return this;
+        },
+
+        a11y_text: function() {
+            Adapt.a11y.log.removed('a11y_text is no longer required. https://tink.uk/understanding-screen-reader-interaction-modes/');
+            return this;
+        },
+
+        a11y_selected: function() {
+            Adapt.a11y.log.removed('$("..").a11y_selected is removed. Please use aria-live instead.');
+            return this;
+        },
+
+        a11y_on: function(isOn) {
+            Adapt.a11y.log.deprecated('$("..").a11y_on, use Adapt.a11y.findTabbable($element); and Adapt.a11y.toggleAccessible($elements, isAccessible); instead.');
```

Comment:
Is there any way of giving more detail about what's calling the deprecated function? At the moment you get a lot of warnings in the console but no detail on where to look to track down the source of the warning...
![image](https://user-images.githubusercontent.com/1249597/62937290-17ad8c00-bdc4-11e9-9a4e-0af5cbcb2654.png)

Reply:
Use a browser which outputs the callstack. I'm afraid you can't change the call stack output to the console without logging or throwing from the original location.

![image](https://user-images.githubusercontent.com/7974663/62937571-ec776c80-bdc4-11e9-8ac9-b83ca700dea2.png)

Or we could output a separate call stack [console.trace()](https://developer.mozilla.org/en-US/docs/Web/API/Console/trace)? It'll really pollute the console output though.

Reply:
erm that's weird I tried that in all browsers (other than Safari). the above screenshot was from Chrome. maybe I'm missing a setting?

Reply:
Nope, can't see any setting to change this... weird.

Reply:
![image](https://user-images.githubusercontent.com/7974663/62938968-0c5c5f80-bdc8-11e9-9dae-9526f30faaa3.png)

Windows 10

Reply:
[console.groupCollapsed](https://developer.mozilla.org/en-US/docs/Web/API/Console/groupCollapsed) + [console.trace](https://developer.mozilla.org/en-US/docs/Web/API/Console/trace) could work?

Reply:
yeah I'm on the same version of Chrome... I can't figure it out at all, it's most annoying as obviously that would be very useful! It's it's just some quirk of my computer then let's leave it; after all once all the core plugins have been updated it should be much easier to track down purely by searching the code for that function name.